connector/spanmetricsconnector/README.md

```diff
@@ -19,31 +19,31 @@ Aggregates Request, Error and Duration (R.E.D) metrics from span data.
 **Request** counts are computed as the number of spans seen per unique set of dimensions, including Errors.
 For example, the following metric shows 142 calls:
 ```
-calls_total{http_method="GET",http_status_code="200",operation="/Address",service_name="shippingservice",span_kind="SPAN_KIND_SERVER",status_code="STATUS_CODE_UNSET"} 142
+calls_total{http_method="GET",http_status_code="200",span_name="/Address",service_name="shippingservice",span_kind="SPAN_KIND_SERVER",status_code="STATUS_CODE_UNSET"} 142
 ```
-Multiple metrics can be aggregated if, for instance, a user wishes to view call counts just on `service_name` and `operation`.
+Multiple metrics can be aggregated if, for instance, a user wishes to view call counts just on `service_name` and `span_name`.
```

Comment:
It should probably be:

```suggestion
Multiple metrics can be aggregated if, for instance, a user wishes to view call counts just on `service.name` and `span.name`.
```

Reply:
I completely agree. We should raise a separate PR to update the documentation.

Reply:
This definitely needs to be modified in the end. We can temporarily ignore the issue of compliance here. @kovrus What do you think

Reply:
Yep, let's follow up on it later.